Compiler warnings from GCC 13

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Compiling master gives compiler warnings. I _think_ this started happening after I updated GCC, but it's hard to test at this point.

I'm using GCC 13.1.1 20230429 for these results.

```
source/PlayerInfoPanel.cpp: In member function ‘void PlayerInfoPanel::DrawPlayer(const Rectangle&)’:
source/PlayerInfoPanel.cpp:611:23: warning: possibly dangling reference to a temporary [-Wdangling-reference]
  611 |         const string &combatRating = GameData::Rating("combat", combatLevel);
      |                       ^~~~~~~~~~~~
source/PlayerInfoPanel.cpp:611:54: note: the temporary was destroyed at the end of the full expression ‘GameData::Rating(std::__cxx11::basic_string<char>(((const char*)"combat"), std::allocator<char>()), combatLevel)’
  611 |         const string &combatRating = GameData::Rating("combat", combatLevel);
      |                                      ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~
source/PlayerInfoPanel.cpp:635:23: warning: possibly dangling reference to a temporary [-Wdangling-reference]
  635 |         const string &attractionRating = GameData::Rating("cargo attractiveness", attractionLevel);
      |                       ^~~~~~~~~~~~~~~~
source/PlayerInfoPanel.cpp:635:58: note: the temporary was destroyed at the end of the full expression ‘GameData::Rating(std::__cxx11::basic_string<char>(((const char*)"cargo attractiveness"), std::allocator<char>()), (int)attractionLevel)’
  635 |         const string &attractionRating = GameData::Rating("cargo attractiveness", attractionLevel);
      |                                          ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
source/PlayerInfoPanel.cpp:636:23: warning: possibly dangling reference to a temporary [-Wdangling-reference]
  636 |         const string &deterrenceRating = GameData::Rating("armament deterrence", deterrenceLevel);
      |                       ^~~~~~~~~~~~~~~~
source/PlayerInfoPanel.cpp:636:58: note: the temporary was destroyed at the end of the full expression ‘GameData::Rating(std::__cxx11::basic_string<char>(((const char*)"armament deterrence"), std::allocator<char>()), (int)deterrenceLevel)’
  636 |         const string &deterrenceRating = GameData::Rating("armament deterrence", deterrenceLevel);
      |                                          ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

### Steps to Reproduce

The birds and the bees.

### Expected Behavior

No warnings.

### Screenshots

_No response_

### Link to save file

_No response_

### Operating System

EndeavourOS (kernel: Linux 6.3.1-zen2-1-zen)

### Game Source

Built from source

### Game Version

master

### Additional Information

_No response_

--- a/source/PlayerInfoPanel.cpp
+++ b/source/PlayerInfoPanel.cpp
@@ -608,7 +608,7 @@ void PlayerInfoPanel::DrawPlayer(const Rectangle &bounds)
 	// Determine the player's combat rating.
 	int combatExperience = player.Conditions().Get("combat rating");
 	int combatLevel = log(max<int64_t>(1, combatExperience));
-	const string &combatRating = GameData::Rating("combat", combatLevel);
+	string combatRating = GameData::Rating("combat", combatLevel);
 	if(!combatRating.empty())
 	{
 		table.DrawGap(10);
@@ -632,8 +632,8 @@ void PlayerInfoPanel::DrawPlayer(const Rectangle &bounds)
 	auto factors = player.RaidFleetFactors();
 	double attractionLevel = max(0., log2(max(factors.first, 0.)));
 	double deterrenceLevel = max(0., log2(max(factors.second, 0.)));
-	const string &attractionRating = GameData::Rating("cargo attractiveness", attractionLevel);
-	const string &deterrenceRating = GameData::Rating("armament deterrence", deterrenceLevel);
+	string attractionRating = GameData::Rating("cargo attractiveness", attractionLevel);
+	string deterrenceRating = GameData::Rating("armament deterrence", deterrenceLevel);
 	if(!attractionRating.empty() && !deterrenceRating.empty())
 	{
 		double attraction = max(0., min(1., .005 * (factors.first - factors.second - 2.)));

